src/createTippy.ts

```diff
@@ -312,6 +312,8 @@ export default function createTippy(
       ) {
         return;
       }
+    } else {
+      instance.props.onClickOutside(instance, event);
     }
```

Comment:
What was the difference between not using the `else` branch and using it? (I noticed you said you fixed a problem with the demo, just trying to get full context)

Reply:
That was the issue, I wasn't using it.
The difference is that if you aren't using the `else` it will be triggered also when you click on the button that opens the tippy. And it will close and reopen just after the tippy so you won't be able to close the Tippy by clicking again on the opener.